web/css/style.css

```diff
@@ -236,6 +244,17 @@ th, td {
 input#docsearch {
   width: 260px;
   padding: 4px 4px 2px 4px;
+  box-shadow: 0px 5px 6px -2px rgb(250,250,250);
```

Comment:
you should use hex colors. They make the CSS smaller.

Reply:
Why not. Done!